apps/learner/src/lib/components/Button/Button.svelte

```diff
@@ -56,11 +65,13 @@
     aria-disabled={disabled}
```

Comment:
Do we still need `aria-disabled` in this since `<button>` natively supports the `disabled` attribute?

Reply:
Looks like this is handled in your next PR in draft.